pkg/innerring/innerring.go

```diff
@@ -121,6 +121,18 @@ func (s *Server) Start(ctx context.Context, intError chan<- error) error {
 	go s.morphListener.ListenWithError(ctx, morphErr)      // listen for neo:morph events
 	go s.mainnetListener.ListenWithError(ctx, mainnnetErr) // listen for neo:mainnet events
 
+	s.morphListener.RegisterBlockHandler(func(b *block.Block) {
+		s.log.Info("new block",
```

Comment:
Let's use debug level for new block messages.

Reply:
Done.